doc/cephfs/disaster-recovery.rst

```diff
@@ -59,3 +59,53 @@ Note that this command acts as a normal CephFS client to find all the
 files in the file system and read their layouts, so the MDS must be
 up and running.
 
+Using first-damage.py
+---------------------
+
+#. Unmount all clients.
+
+#. Flush the journal if possible:
+
+   .. prompt:: bash #
+      
+      ceph tell mds.<fs_name>:0 flush journal
+
+#. Fail the file system:
+
+   .. prompt:: bash #
+
+      ceph fs fail <fs_name>
+
+#. Recover dentries from the journal. If the MDS flushed the journal
+   successfully, this will be a no-op:
+
+   .. prompt:: bash #
+
+      cephfs-journal-tool --rank=<fs_name>:0 event recover_dentries summary
+
+#. Reset the journal:
+   
+   .. prompt:: bash #
+
+      cephfs-journal-tool --rank=<fs_name>:0 journal reset --yes-i-really-mean-it
+
+#. Run ``first-damage.py`` to see the list of damaged dentries:
+
+   .. prompt:: bash #
+
+      python3 first-damage.py --memo run.1 <pool>
+
+#. Optionally, remove the damaged dentries:
```

Comment:
s/the//

Reply:
I thought about this for two hours. I think that the definite article makes the sentence more natural. I'm leaving it in for now.